operate/client/src/modules/utils/flowNodeSelection.test.ts

```diff
@@ -114,3 +117,130 @@ describe('getSelectedRunningInstanceCount', () => {
     expect(result).toBe('Start Event');
   });
 });
+
+describe('selectAdHocSubProcessInnerInstance', () => {
```

Comment:
Please avoid nesting tests. I know it's inconsistent on the codebase but we're trying to improve it little by little.

Reasoning can be [found here](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing)